content/en/cosign/hardware-based-tokens.md

```diff
@@ -6,23 +6,30 @@ position: 114
 
 The `cosign` command line tool optionally supports hardware tokens for signing and key management.
 This support is enabled through the [PIV protocol](https://csrc.nist.gov/projects/piv/piv-standards-and-supporting-documentation)
-and the [go-piv](https://github.com/go-piv/piv-go) library, which is not included in the standard release. Use [`make cosign-pivkey`](https://github.com/sigstore/cosign/blob/a8d1cc1132d4a019a62ff515b9375c8c5b98a5c5/Makefile#L52), or `go build -tags=pivkey`, to build `cosign` with support for hardware tokens.
+and the [go-piv](https://github.com/go-piv/piv-go) library, which is not included in the standard release. Use `make cosign-pivkey-pkcs11key`, or `go build -tags=pivkey,pkcs11key ./cmd/cosign`, to build `cosign` with support for hardware tokens.
 
 ---
 **NOTE**
 
-Cosign's hardware token support requires `libpcsclite` on platforms other than Windows and OSX.
+`cosign`'s hardware token support requires `libpcsclite` on platforms other than Windows and OSX.
```

Comment:
I would revert this too as it seems to be talking about the tool rather than the command